swagger_server/ogm/utils.py

```diff
@@ -0,0 +1,28 @@
+def with_retry(exception_class, max_retries=3):
+    def outer_wrapper(method):
+        def wrapper(instance, *args, **kwargs):
```

Comment:
```suggestion
        def wrapper(instance, *args, **kwargs):
            count = 0
            while count < max_retries:
                try:
                    return method(instance, *args, **kwargs)
                except exception_class:
                    count += 1
            return method(instance, *args, **kwargs)
```

Reply:
I was going for the Decorator Factory pattern: https://www.artima.com/weblogs/viewpost.jsp?thread=240845#decorator-functions-with-decorator-arguments